### ISSUE TITLE
Seeking contributors?

### ISSUE THREAD

**Issue by justinmaurerdotdev** [chromatix-app/chromatix-release#1]
Hi there, I just stumbled on a reference to this project on Reddit, and it looks great. It's a project that I'm quite interested in, and, if it's a JavaScript project, may be able to actually help with. Is there any plan to make the code open source so that others in the community can contribute to its development, or to bring other developers in?

**Comment by yznts** [comment]
Yup, lack of open code makes me a little nervous. There is a nice iOS app named "Prism Music" that was silently abandoned despite requests of updates or open-sourcing it.

**Comment by yznts** [comment]
I'd help with non-plex platforms integration like Jellyfin

**Comment by alexb148** [comment]
Thanks for your interest. At the moment I'm not looking for contributors, it's just a personal project, but maybe down the line.

I do plan to open source the code base at some point. I've now open-sourced the Electron wrapper that is used for building desktop apps, but not the main web app itself. It's a bit of a mess and needs a lot of tidying up and reworking.

Search and an "all songs" view are quite high up my to-do list, so hopefully soon.

I think things like ripping CD's, converting tracks, and purchasing from stores are unlikely to happen as this is quite complex functionality that goes far beyond simply integrating with the Plex API.

Potentially I'd like to integrate with other services like Jellyfin and even Spotify, but that's probably some way off, and will require quite a bit of reworking first.

**Comment by yznts** [comment]
Sure, no pressing :)
Just want to be sure you know we are around and tracking current/future state of things.
If you really plan to open-source it at some point, it's great! Even without seeking contributors, it's a nice move for community. In that way we are sure project stays alive even if you'll decide to stop development (yes, I'm talking about you, Prism Music). Burnout, loss of interest, etc, it happened to many of us.
If you'll decide to go another way and monetize, it still a good option as long as you support the project.

After working with lots of corp code, I don't believe your words about mess :)

As an example, I have an issue with playing some albums from my library (not playing at all) and want to debug it properly before creating any issue. Probably related to formats, not sure. For now I'm just skipping them.
